### ISSUE TITLE
Cannot Monitor Email Opened, Clicked Linked and Submitted Data

### ISSUE THREAD

**Issue by shvmsntsh** [gophish/gophish#1695]
**What version of Gophish are you using?:** 
gophish-v0.8.0-windows-64bit.zip

**Brief description of the issue:** 
I am using IIS to open my port 4300 to public where I have added add a custom page to test and capture submitted data.

**What are you expecting to see happen? :**
I think I am doing everything right and I even successfully tracked email clicks and so on in a previous version of GoPhish a few months back but I am unable to right now no matter what I try

What are you seeing happen?
![image](https://user-images.githubusercontent.com/23379319/70994726-d7a14180-20f4-11ea-9636-08910f58d139.png)

**Here is my Landing Page. I am testing with an html code of old gmail sigin page**
<!DOCTYPE html><html lang="en"><head>
	<base href="http://100.37.134.38:4300/"/>
	<title></title>
	<meta charset="utf-8"/><meta name="viewport" content="width=device-width, initial-scale=1"/>
	<link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet"/><script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script><script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
	<style type="text/css">body{
		padding: 0;
		margin: 0;
	}
	h1{
		text-align: center;
		padding: 5px 0;
	}
	img{
		width: 15%;
		display: block;
		margin: 0 auto;
		padding-top: 40px;
		padding-bottom: 40px;
	}
	#w3hubs{
		margin: 30px;
	}
	form{
		width: 70%;
		display: block;
		margin: auto;

	}
	.bg-light{
		padding: 45px;
		-webkit-box-shadow: 0 8px 6px -6px #ddd;
       -moz-box-shadow: 0 8px 6px -6px #ddd;
            box-shadow: 0 8px 6px -6px #ddd;

	}
	.form-group{
		margin: 0;

	}
	.form-control{
		padding: 1.875rem 1rem;
	}
	.btn{
		margin-top: 10px;
		width: 100%;
	}
	.form-check{
		margin-top: 15px;
	}
	.btn-link:hover{
		text-decoration: none;
	}
	h2{
		text-align: center;
		padding: 30px 0;
		font-size: 42px;
	}
@media(max-width: 768px){
	form{
		width: 90%
	}
	img{
		width: 40%;
		padding-top: 0px;
		padding-bottom: 10px;
	}
}
@media(max-width: 475px){
	.form-control{
		padding: 1.4em 
	}
	form{
		width: 100%;
	}
	.bg-light{
		padding: 30px;
	}
	h1{
		padding: 10px 0;
	}
	.btn-link{
		text-align: left !important;
	}
}
	</style>
</head>
<body>
<section id="w3hubs">
<div class="container">
<h2><span class="text-primary">G</span><span class="text-danger">o</span><span class="text-warning">o</span><span class="text-primary">g</span><span class="text-success">l</span><span class="text-danger">e</span></h2>

<h1>One Account.All Of Google.</h1>

<h3 class="text-center pb-2">Sign in to continue to Gmail</h3>

<div class="bg-light"><img class="img-circle" src="https://cdn3.iconfinder.com/data/icons/login-5/512/LOGIN_6-512.png"/>
<form action="" method="post"><input name="__original_url" type="hidden" value="https://accounts.google.com"/>
<div class="form-group"><input class="form-control" name="" placeholder="Email" type="text"/></div>

<div class="form-group"><input class="form-control" name="password" placeholder="Password" type="password"/></div>
<button class="btn btn-primary btn-lg" onclick="window.location.href = &#39;https://accounts.google.com&#39;;" type="button">Sign In</button>

<div class="row">
<div class="col-md-6 col-xs-12">
<div class="form-group form-check"><label class="form-check-label"><input class="form-check-input" type="checkbox"/> Stay Signed In </label></div>
</div>

<div class="col-md-6 col-xs-12"><a class="text-right btn btn-link" href="#">Need help ?</a></div>
</div>
</form>
</div>
<a class="btn btn-link" href="#">Create an account</a></div>
</section>

</body></html>

**Campaign:**
![image](https://user-images.githubusercontent.com/23379319/70994519-74171400-20f4-11ea-899f-934a898f4d00.png)

Terminal Output after successfully sending a Campaign
![image](https://user-images.githubusercontent.com/23379319/70994622-a6c10c80-20f4-11ea-8044-7724247ea5fc.png)

This is my config.json -
{
	"admin_server": {
		"listen_url": "0.0.0.0:3333",
		"use_tls": true,
		"cert_path": "gophish_admin.crt",
		"key_path": "gophish_admin.key"
	},
	"phish_server": {
		"listen_url": "0.0.0.0:4300",
		"use_tls": false,
		"cert_path": "example.crt",
		"key_path": "example.key"
	},
	"db_name": "sqlite3",
	"db_path": "gophish.db",
	"migrations_prefix": "db/db_",
	"contact_address": "",
	"logging": {
		"filename": ""
	}
}

**Comment by jordan-wright** [comment]
The issue is likely that IIS is handling the connection instead of Gophish. In order to capture events, connections need to be handled by the Gophish web server.

**Comment by shvmsntsh** [comment]
Thank you for the response.

But then how can we make the port 4300 as mentioned in config.json accessible to the email receiver on clicking on links?

**Comment by jordan-wright** [comment]
You either need to expose that port to the internet so it receives connections directly or, if you must use IIS, it should only proxy the connection back to an internal Gophish instance.
